cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/monitor/LoadMonitor.java

```diff
@@ -107,11 +108,13 @@
    *
    * @param config The load monitor configuration.
    * @param time   The time object.
+   * @param executor The proposal executor.
    * @param dropwizardMetricRegistry The sensor registry for cruise control
    * @param metricDef The metric definitions.
    */
   public LoadMonitor(KafkaCruiseControlConfig config,
                      Time time,
+                     Executor executor,
```

Comment:
I suspect that we can limit ourselves to passing only the state (not the whole `executor`) -- please see the other comment above.